packages/amplication-client/src/Resource/git/dialogs/GitCreateRepo/GitCreateRepo.tsx

```diff
@@ -108,6 +127,27 @@ export default function GitCreateRepo({
             showError={false}
           />
 
+          {!!values.name && values.name.length > 1 && (
+            <FlexItem
+              direction={EnumFlexDirection.Column}
+              gap={EnumGapSize.Default}
+            >
+              <Text
+                textStyle={EnumTextStyle.Subtle}
+                textColor={EnumTextColor.ThemeGreen}
+              >
+                {"Your new repository will be created as "}
+                <b>{processGitRepositoryName(values.name)}.</b>
```

Comment:
use EnumTextWeight instead of <b> (or just remove it entirely, maybe we don't need to add bold and color together)